### ISSUE TITLE
Installing nix with quickstart.sh is dangerous

### ISSUE THREAD

**Issue by michael-yx-wu** [CRogers/baconsnake#4]
It's not a good idea to assume people want to install nix. Chances are users already have homebrew installed. At the very least, you should make it clear that the startup script is going to be installing some heavy stuff in the form of nix. An alternative would be to turn the startup script into some basic instructions like "install node (or nodenv) w/ your favorite package manager"

**Comment by CRogers** [comment]
Nix is not dangerous (that's the whole point of it). I usually mention that people can use the webpack script directly after an npm install, but we've had problems with people having the wrong version of node, hence the use of nix. Also happy from someone to replace nix with downloading a specific version of node and npm in a way that won't conflict with people's normal setups (similar to nix) and using that.

**Comment by jmcampanini** [comment]
it asks for sudo :(

**Comment by jmcampanini** [comment]
there's also a *"fantastic"* gradle plugin that will download a specific version of node + npm

repo: https://github.com/srs/gradle-node-plugin

**Comment by michael-yx-wu** [comment]
Probably not relevant here since this project doesn't need gradle. Personally the right thing feels like encouraging people to just use nodenv with whatever package manager they already have.

**Comment by jmcampanini** [comment]
> Probably not relevant here since this project doesn't need gradle.

agreed. but they didn't use nix until it was added. gradle isn't only for just compiling JVM code... ;) but that's a separate convo.

>  just use nodenv with whatever package manager they already have.

+1 and specifying a specific minimum version of npm/node seems reasonable
